scipy/interpolate/interpolate.py

```diff
@@ -2801,30 +2799,30 @@ def append_func(mk):
             sl_0, sl_N = conds
 
         # Now set up the (N+1)x(N+1) system of equations
-        beta = np.r_[0,2*(xk[2:]-xk[:-2]),0]
-        alpha = xk[1:]-xk[:-1]
-        gamma = np.r_[0,alpha[1:]]
-        B = np.diag(alpha,k=-1) + np.diag(beta) + np.diag(gamma,k=1)
+        beta = np.r_[0, 2*(xk[2:]-xk[:-2]), 0]
+        alpha = xk[1:] - xk[:-1]
+        gamma = np.r_[0, alpha[1:]]
+        B = np.diag(alpha, k=-1) + np.diag(beta) + np.diag(gamma, k=1)
         d1 = alpha[0]
         dN = alpha[-1]
         if kind == 'not-a-knot':
             d2 = alpha[1]
             dN1 = alpha[-2]
-            B[0,:3] = [d2,-d1-d2,d1]
-            B[-1,-3:] = [dN,-dN1-dN,dN1]
+            B[0, :3] = [d2, -d1 - d2, d1]
+            B[-1, -3:] = [dN, -dN1 - dN, dN1]
         elif kind == 'runout':
-            B[0,:3] = [1,-2,1]
-            B[-1,-3:] = [1,-2,1]
+            B[0, :3] = [1, -2, 1]
+            B[-1, -3:] = [1, -2, 1]
         elif kind == 'parabolic':
-            B[0,:2] = [1,-1]
-            B[-1,-2:] = [-1,1]
+            B[0, :2] = [1, -1]
+            B[-1, -2:] = [-1, 1]
         elif kind == 'periodic':
-            raise NotImplementedError
+            raise NotImplementedError("""""")
```

Comment:
I think one uses `""` instead of multi-line `""""""`comments, especially if there is no message.